posts/2021-09-03-210010-beta.adoc

```diff
@@ -0,0 +1,237 @@
+* link:https://openjdk.java.net/jeps/403[Strongly Encapsulate JDK Internals]
+* link:https://openjdk.java.net/jeps/409[Sealed Classes]
+* link:https://openjdk.java.net/jeps/411[Deprecate the Security Manager for Removal]
+
+By taking advantage of trying out the new changes in Java 17 now, you get more time to review your applications, microservices and runtime environments and be a step ahead when it becomes generally available.
+
+To give this a try today, download the link:https://jdk.java.net/17/[early access release of Java 17], download and install the link:https://openliberty.io/downloads/#runtime_betas[21.0.0.10-beta] version of Open Liberty, edit your Liberty server's link:https://openliberty.io/docs/21.0.0.8/reference/config/server-configuration-overview.html#server-env[server.env file] to point JAVA_HOME to your Java 17 installation and start testing!
+
+For more information on Java 17, please visit the Java 17 link:https://jdk.java.net/17/release-notes[release notes page], link:https://download.java.net/java/early_access/jdk17/docs/api/[API Javadoc page] or link:https://jdk.java.net/17/[download page].
+For more information on Open Liberty, please visit our link:https://openliberty.io/docs/21.0.0.8/overview.html[documentation page].
+
+*Note: As we work towards full Java 17 support ourselves, please excuse any of our functionality that might not be 100% ready yet.
+
+
+[#mp_cp]
+=== MicroProfile Context Propagation 1.3
+
+MicroProfile Context Propagation 1.3 Release Candidate 1 adds support for Jakarta EE 9 packages (`jakarta.*`) and eventual alignment with the MicroProfile 5.0 platform.
+With the 1.3 RC1 release, you can now use MicroProfile Context Propagation within your Jakarta EE 9 applications.
+
+To enable the new beta features in your app, add them to your `server.xml`:
+
+[source, xml]
+----
+<server>
+  <featureManager>
+    <feature>cdi-3.0</feature>
+    <feature>jndi-1.0</feature>
+    <feature>mpContextPropagation-1.3</feature>  
+    <feature>servlet-5.0</feature>
+  </featureManager>
+</server>
+----
+
+If you're using link:{url-prefix}/guides/maven-intro.html[Maven], here are the coordinates:
+
+[source, xml]
+----
+<dependency>
+    <groupId>org.eclipse.microprofile.context-propagation</groupId>
+    <artifactId>microprofile-context-propagation-api</artifactId>
+    <version>1.3-RC1</version>
+</dependency>
+----
+
+For more information on this MicroProfile Context Propagation update, refer to the link:https://download.eclipse.org/microprofile/microprofile-context-propagation-1.3-RC1/apidocs/[JavaDoc] and link:https://download.eclipse.org/microprofile/microprofile-context-propagation-1.3-RC1/microprofile-context-propagation-spec-1.3-RC1.html[specification].
+
+[#value_add]
+=== Jakarta EE 9 support for the remaining Open Liberty value-add features
+
+In Open Liberty 21.0.0.10-beta we now have Jakarta EE 9 support for the following value-add features:
+
+* Admin Center (adminCenter-1.0)
```

Comment:
make each of these link to their corresponding feature in docs. For example 
Admin Center ([adminCenter-1.0]({url-prefix}/docs/latest/reference/feature/adminCenter-1.0.html))